docs/en/ingest-management/agent-policies.asciidoc

```diff
@@ -273,3 +273,9 @@ Make sure that the {agent}s assigned to this policy all have connectivity to the
 that you added. Lack of connectivity will prevent the {agent}
 from checking in with the {fleet-server} and receiving policy updates, but the agents
 will still forward data to the cluster.
+
+[discrete]
+[[agent-policy-scale]]
+== Scale of Policies Supported
```

Comment:
I have a few thoughts about this section:

* "Scale of policies supported" doesn't seem intuitive unless you think users will search on this phrase. Maybe replace with "Policy limits" or "Policy scaling recommendations"?
* I think it would be better to have this section appear earlier in the topic, maybe under the section about policy benefits?

Reply:
agree on both points yes. it is a recommendation indeed.